### DISCUSSION TITLE
Bookmarks

### DISCUSSION THREAD

**Discussion by jman-schief** [FossifyOrg/General-Discussion#573] [Ideas]
Hi!

I often listen to long podcasts (e.g. 2 hours) in my commute to work, so it'll sometimes take multiple listening "sessions" to reach the end. Sometimes these sessions are spaced in time so I forget where I left a podcast a few days ago. If I close the app or do other actions I lose the point where I left the podcast.

Being able to persist a bookmark would be useful for these cases. Is there anything I can use for that?

Thanks

**Comment by Aga-C** [comment]
Please create a feature request for it.
